src/main/java/frc/robot/constants/DeployingBarConstants.java

```diff
@@ -0,0 +1,10 @@
+package frc.robot.constants;
+
+public class DeployingBarConstants {
+  public static final int kMotor = 0;
+  public static final double kMaxRotation = 90, kMinRotation = 0;
+  public static final double kP = 1, kI = 0, kD = 0;
```

Comment:
generealy we put constants on diffrent lines

Reply:
There are gonna be some pretty drastic code changes since they are now using limit switches, so I'd recommend looking at this branch later